components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -1474,7 +1474,7 @@ spec:
             secretKeyRef:
               key: db.name
               name: tekton-results-database
-        image: quay.io/redhat-appstudio/tekton-results-api:ed360eccc021ad5eedf8ea9c0732912ef602b15a
```

Comment:
cool this is what we want @enarha for SRVKP-5832 @enarha 

see https://quay.io/repository/konflux-ci/tekton-results-api?tab=tags